pkg/lockfile/ecosystems_test.go

```diff
@@ -35,7 +36,7 @@ func TestKnownEcosystems(t *testing.T) {
 
 	// npm, yarn, and pnpm, and pip and poetry, all use the same ecosystem
 	// so "ignore" those parsers in the count
```

Comment:
This comment should be updated:

```suggestion
	// npm, yarn, and pnpm, and pip and poetry, and maven and gradle, all
	// use the same ecosystem so "ignore" those parsers in the count
```